pkg/metrics/metrics.go

```diff
@@ -29,10 +29,13 @@ var (
 		"The number of errors in the reconciler",
 		stats.UnitDimensionless)
 
-	// PipelineError metric measures the error by components when syncing a commit
+	// PipelineError metric measures the error by components when syncing a commit.
+	// Definition here must exactly match the definition in the resource-group
+	// controller, or the Prometheus exporter will error. b/247516388
+	// https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/metrics/metrics.go#L35
```

Comment:
Shouldn't we point to the resource-group controller metric text (https://github.com/GoogleContainerTools/kpt-resource-group/blob/ede29f6f10dca881ffe63b86acf5ca0e23ba6c5f/controllers/metrics/metrics.go#L88) instead?

Reply:
lol oops. good catch